lib/index.js

```diff
@@ -14,7 +14,7 @@ const AWS_SECRET_ACCESS_KEY = 'test';
 const getLocalEndpoint = () => {
   const port = process.env.EDGE_PORT || DEFAULT_EDGE_PORT;
   const host = process.env.LOCALSTACK_HOSTNAME || DEFAULT_HOSTNAME;
-  return `http://${host}:${port}`;
+  return `https://${host}:${port}`;
```

Comment:
I think we should introduce a way to override the protocol to be plain `http`, as some users may be facing issues downloading our SSL certificate. 

In fact, might be the easiest if we add a `LOCALSTACK_ENDPOINT` env variable that allows to optionally customize the endpoint URL to connect to (let's also add a hint in the README for it 👍 )